cmd/monaco/integrationtest/v2/test-resources/integration-all-configs/project/openpipeline/events.json

```diff
@@ -0,0 +1,88 @@
+{
+    "customBasePath": "/platform/ingest/custom/events",
+    "editable": true,
+    "endpoints": [
+        {
+            "basePath": "/platform/ingest/v1/events",
+            "builtin": true,
+            "defaultBucket": null,
+            "displayName": "Default API",
+            "editable": false,
+            "enabled": true,
+            "processors": [],
+            "routing": {
+                "type": "dynamic"
+            },
+            "segment": ""
+        }
+    ],
+    "id": "events",
+    "pipelines": [
+        {
+            "builtin": true,
+            "dataExtraction": {
+                "editable": false,
+                "processors": []
+            },
+            "displayName": "events",
+            "editable": false,
+            "enabled": true,
+            "id": "default",
+            "metricExtraction": {
+                "editable": false,
+                "processors": []
+            },
+            "processing": {
+                "editable": false,
+                "processors": []
+            },
+            "securityContext": {
+                "editable": false,
+                "processors": []
+            },
+            "storage": {
+                "catchAllBucketName": "default_events",
+                "editable": false,
+                "processors": []
+            },
+            "type": "default"
+        },
+        {
```

Comment:
Adding a second pipeline to make this a non-default config is a good idea, but considering we can't delete (or cleanup) configurations, how do we track whether changes are really being accepted by the API?

Reply:
This would require a separate/dedicated e2e test which does the required assertions, we can think about adding one (in a follow up) :)